lib/plugins/rotate_asg_instances/asg.rb

```diff
@@ -1,20 +1,36 @@
-require 'moonshot/ssh_fork_executor'
-
 module Moonshot
   class ASG # rubocop:disable Metrics/ClassLength
     include Moonshot::CredsHelper
 
-    def initialize(resources)
+    def initialize(resources, ssh_user)
       @resources = resources
       @ilog = @resources.ilog
-      @ssh_user = ENV['MOONSHOT_SSH_USER'] || ENV['LOGNAME']
```

Comment:
Here we are removing the ability to use `ENV['LOGNAME']` but not adding it anywhere else, is it ok to lose this feature?

Reply:
I think we should specify LOGNAME along with CDB_CI_USER in the worker PR where this plugin is configured.